crates/sui-indexer/src/store/module_resolver.rs

```diff
@@ -25,8 +25,8 @@ impl IndexerModuleResolver {
 }
 
 const LATEST_MODULE_QUERY: &str = "SELECT (t2.module).data
-FROM (SELECT UNNEST(data) AS module
-      FROM (SELECT data FROM packages WHERE package_id = $1 ORDER BY version DESC FETCH FIRST 1 ROW ONLY) t1) t2
+FROM (SELECT UNNEST(modules) AS module
```

Comment:
do we plan to alter the table in v1?

Reply:
no? why do we want to do that?

btw I think the package table is not complete yet, we still need to figure out how to do package upgrade